### PR TITLE
bug fix in basic client

### PR DESCRIPTION
**Pull request by yc7z**
# PR Type
[Fix]

Bug fix in `basic_client.py`

Clickup Ticket(s): Link(s) if applicable.

# Tests Added

`test_setup_client_basic`

### CODE REVIEW

File: fl4health/clients/basic_client.py [hunk 1]
```diff
@@ -557,7 +557,7 @@ def setup_client(self, config: Config) -> None:
 
         self.wandb_reporter = ClientWandBReporter.from_config(self.client_name, config)
 
-        self.intialized = True
+        self.initialized = True
```
Comment (yc7z) [inline fl4health/clients/basic_client.py:560]: This was the bug

Comment (emersodb) [reply fl4health/clients/basic_client.py:560]: Good catch. That is definitely a problem. I guess we don't have a test covering this. As part of this. Do you mind added a test that confirms that this is properly set after `setup_client.py` is run?

**Review by emersodb — APPROVED** [review]
Other than adding the test to cover the case that was missed here, we're good to go.

**Review by jewelltaylor — APPROVED** [review]
The added test looks good! Makes sure the code wont regress to make similar errors in the future :)

**Pull request merged**